packages/agoric-cli/src/start.js

```diff
@@ -284,26 +284,48 @@ export default async function startMain(progname, rawArgs, powers, opts) {
 
     // Get or create the essential addresses.
     const addrs = {};
-    for (const keyName of ['provision', 'delegate0']) {
+    for (const keyName of ['provision', 'delegate0', 'rly2']) {
```

Comment:
I'd like it if you could separate these change to agoric-cli into a different PR, and just make an option to allow recovering a seed phrase.  It's definitely something that would be useful to merge.